google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -870,6 +870,37 @@ def initialize gapi
             @schema = nil
           end
 
+          ##
+          # Returns the table's schema. This method can also be used to set,
+          # replace, or add to the schema by passing a block. See {Schema} for
+          # available methods.
+          #
+          # @param [Boolean] replace Whether to replace the existing schema with
+          #   the new schema. If `true`, the fields will replace the existing
+          #   schema. If `false`, the fields will be added to the existing
+          #   schema. When a table already contains data, schema changes must be
+          #   additive. Thus, the default value is `false`.
+          # @yield [schema] a block for setting the schema
+          # @yieldparam [Schema] schema the object accepting the schema
+          #
+          # @return [Google::Cloud::Bigquery::Schema]
+          #
+          # @example
+          #   require "google/cloud"
+          #
+          #   gcloud = Google::Cloud.new
+          #   bigquery = gcloud.bigquery
+          #   dataset = bigquery.dataset "my_dataset"
+          #   table = dataset.create_table "my_table" do |schema|
```

Comment:
Although this is a great example, I don't see this method (`Updater#schema`) called anywhere in it. Should this example be added to the list on `Dataset#create_table`, and replaced here with a different one?